Misc/NEWS.d/next/Library/2023-05-11-01-07-42.gh-issue-102613.uMsokt.rst

```diff
@@ -0,0 +1,2 @@
+Fix issue where :meth:`pathlib.Path.glob` raised :exc:`RecursionError` when
+walking deep directory trees.
```

Comment:
```suggestion
Fix :meth:`pathlib.Path.glob` raising :exc:`RecursionError` when
walking deep directory trees.
```

Reply:
Hm - your suggestion isn't obviously better to my eyes. Would you mind explaining why you prefer it?

Reply:
It feels to me that "issue where" is obvious from the context. It feels, at least, that the second one reads better.

Reply:
Hmm, I think I agree with Barney here -- the original reads better, to me